src/DynamoCoreWpf/Views/GuidedTour/HtmlPages/learnPackages.en-US.html

```diff
@@ -0,0 +1,39 @@
RCaqJvaKDPLNYnNC3e6aEpl7elnNBby1lZFkoNRhPrqkZQx6xiPUbyw80cVcNMedcirZUBK+v2His1CSQ7ry5CC5OCKcFOQefASCVTqaZkTT47Td1iFopRQj4ndbXreiA2Nm29ydSTssEjFouNMgsD1aGKWVWGcldriVBzulwuDdYGlKNYpNWRnUREFW1xVPb+iiamNHYBRBNWmu5Sta9FHGbFOi4c1c1sYUBQF5QjRHILqJNaD3SonEIdKZ/9lMubIrpq1bmheMr6CK29yHaXlolGBQiJ45AUK5QICkYFk+JJEQmrd40eVu9ACwOCMq9rLZVCzkra2tC2XHFpXRhRqRV9PdZ/Dnmzyf4z0trzrNmUyde6R/aZ6XK1OZtLF6wvxF+oVXKZdL5YqppjPONsd17dCuqpaWFI0EkyFZVbJyAUA4FIi/3RFdLWWtFIFQNhXy2QCnVGbTAklYA8HQ0tKhhpcZ7E1pO9u8V5FdsjmUuMdt7rtqY9na+KVhfsEAwLxlucZvOsl5szoLdZD+1uM18abEzKG9SQN6khb1FDnk2GEW4Too0P1mKrToM6ZSXWeXtNE15B9wodXiHsFeLKXz0P71zQIwR09+AyFY/MkByPDcX4butfs377W5lXzqWtI8Gc5Gjsb8XVzLWaJD/ZUTvf2TD9fwEEs8n2AAAAAAEAAf//AA942mNgIAp4AaE1gzXTHgYGJn4Ghv8WLFL/FjGd+P+FSfr/l/86IB6Kmjtoqu4gqXMHQnMGc8bc/wkgPmPF/1BkceY1jPNBfObtjFNBNADHmCnuAAB42q1WaXPTVhSVvMVJyFKy0KIuT7w4Te0nk1IIBkwIkmUX3MXZWglKK8VOui/QMsNv0K+5Mu0M/cZP67mSbQxJ2hmmmYzuefcdvXd3mTQlSNv3XF+I9jNtdrtNhd17Hl02aM0PjkS071GmFP5d1IpatysPDNMkzSfNkY2+pmtOYFukKxLBkUUZJXqCnncot3qvv6ZPOW7XpYLrmZQt+Tv3PVOaRuQJ6nSg2vINQTVGNd8XccoOe7QG1WAlaJ3315n5vOMJWBOFgqY6XgCN4L0pRhuMNgIj8H3fIL3i+5K0jnfo+xZllcA5uVIIy/JOx6O8tKkgbfjhkx5YlFMSdolenD+wBe+wxUZqAT8pF7hdypZNbDoiEhEuiNfzJTi57QUdI9zxPembvqCtXQ97Brs2uN+ivKIJp9LXMmmkClhKWyLi0g4pc3BEehdWUL5s0YQSbOq0032W0w4En0Bbgc+UoJGYWlT9iWnNce2yOYr9pHo5F1PpKXoFJsD2UiDcSIaclyRemsExJWHAyKGVyI4MG+kV06e8Tit4SzNeuDb+0hmVONSfnsoi2YY0/bJp0YyKMxmXemHDolkFohB0xrnLrwNI26cZXu1gNYOVRXM4Zj4JiUAEuriXZp1ARIGgWQTNonnV3vPiXK/hr9DMoXxi0Ruqve21d1OlYUK/kOjPqlibc/a9eG7OIT20aa7CNYtKtuMz/JjBg/RlZCJb6ngxBw/e2hHyy9eWTYnXhthI9/kVtAJrfHjSgv0taF9O1SkJjDVtQSJaDmmbfV3Xk1wtKC3WMu6eR3PSFi5No/imJArOFgGu/2t+XtdmNduOgvhsoUKPK8YFhGkRvi1ULFpSsc5yGXFmeU7FWZZvqjjH8i0V51meV3GBpaHiCZZvq7jI8h0VT7L8QMlh3KkQIMJSVEl/wA1iUXlsc3m0+TDdrIxtro42H6Wb7yqNZiqv4d978O9d2CXgH0sT/rG8AP9YSvjHcgX+sSzBP5ar8I/l+/CP5Rr8Y6mUqCdlailcOx8IB7kNnCSVaD3FtVpVZFXIQhdeRAO0xClZlGFN8kT8V4bB3q+PUqsv08VynNeXXA+DjB38cDwyx7cvKXElsfcj8HT3+CXozhMvZ722/KfGf41NWYsv6Uvw6DL8h8En24umCGsWXVHVc3WLNv6LigLugn4VKdGWS6IqWtz4COWdKGrJFiaFhy8EBiumwYauLy3i/hom1DKaC/8JhSadymFUlULUI5x17cW2qKZnUA5ngiUo4Fmxte09zYisMJ5mVrPnfZvnZxGjWCZs2UTnOq+2YcAzLP1cZJygJynrhD1sZ5zQAA54fr36TgiTMNVlEzmUuKEJvyCSW3DeCZfIdFLmMBwQ+zwKKn/sVJzIHpUSI/DspBPyxV1I+XWOgYAmvzqIgawjNDcSNRXRPEI0ZYsv42zVk5CxA4OIanteVdTxdWWLB0rBtgxDXihhdWf8Q54m6qQKHmRGchnfHFjgDFMT8Jf+VReHqdzEPKhy1JoY6nW/Glf1RTTgrZG6M67eepl9Iue2olrlxENtRdcqES7mYoG1xzlIS5WqoDqjChtGl4tLotSraJL0uAaGBub/a5Ri6/+qPjaf50tdYoSM5dv0Bza6HIyh/03235SDAAz8GLncgstLaXPilwH6cKFKl9GLH5+iv4OZqy8u0BXgu4quQrQ5ai7iKpr4DA7j9InicqQ24KeqjzkD8BmAzuBz1dcTTQcg0WwzxwXYYQ6DXeYw2GMOg33m3Ab4gjkMvmQOA485DHzmOAD3mMPgPnMYfMUcBg+Y0wT4mjkMvmEOg4A5DELm2AAHzGHQZQ6DHnMYHCq6PgrzES9oE+jbBN0C+i6pJyy2sPhe0Y0R+wdeJOwfE8TsnxLE1J8V1UfUX3iRUH9NEFN/SxBTHyq6OaI+4kVC/T1BTP0jQUx9rJ5O5jLDH152hYqHlF3pPBl+U6x/ANocY1d42mPw3sGQEBSxkZGxL3ID404OBg6G5IKNDOxOWxkYTBRYGLRAbAcudxZrFnUOUXYWDqiAL5MjkyGbHCtYgNtprxADPwMXA2sDAwsDJ1CE32kvgwMUgkWYU1w2qjB2BEZscOiIAPPUQLxdHA0MjCwOHckhQEEGl0ggcODyZLFl0eQQZ2fh0drB+L91A0vvRiYGl82sKWwMLi4AEI8oBQAAAAAAAQAAAAA=);
```

Comment:
so if you embed pngs directly into the .html statically files then we'll end up wasting a bunch of space on each localized version of the image... there might be another task on this - @QilongTang ?

Reply:
Hi @QilongTang, I agree with @mjkkirschner since we are going to have the same font style and images in the HTMLs. 
Do you think we should create another task for it? 
FYI: @RobertGlobant20

Reply:
I think so @filipeotero Please log it with details

Reply:
Task created:
https://jira.autodesk.com/browse/DYN-4185